src/command/dev/do_dev.rs

```diff
@@ -1,6 +1,6 @@
 use anyhow::{anyhow, Context};
 use camino::Utf8PathBuf;
-use rover_std::Emoji;
+use rover_std::{Emoji, Style};
```

Comment:
would that I could import some style into my life

Reply:
highly recommend